src/framework/preferences/css/stylus/SeparatedPanelPrefsEditorFrame.styl

```diff
@@ -151,10 +151,11 @@ html {
                 .fl-prefsEditor-panel-descr {
                     clip: auto;
```

Comment:
According to [CSS clip property usage at w3schools](https://www.w3schools.com/cssref/pr_pos_clip.asp): `Note: The clip property does not work if "overflow:visible".` The use of `clip: auto` is unnecessary. 

Also wondering if the use of `overflow: visible` is needed. If not specified, the inherited `hidden` value seems good too.